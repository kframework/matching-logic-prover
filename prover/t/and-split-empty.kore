claim \and()
strategy and-split
