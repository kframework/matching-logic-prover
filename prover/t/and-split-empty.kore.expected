<prover>
  <exit-code>
    0
  </exit-code>
  <goals>
    <goal>
      id: ., parent: .
      <claim>
        \and ( .Patterns )
      </claim>
      <k>
        .
      </k>n<expected>
        .
      </expected>
      <local-context>
        .LocalDeclCellSet
      </local-context>
    <trace>
      .
    </trace>
  </goals>
  <declarations>
    <declaration>
      axiom ax0 : \and ( .Patterns )
    </declaration>
  </declarations>
</prover>
